alacritty/src/config/monitor.rs

```diff
@@ -21,6 +21,11 @@ pub fn watch(mut paths: Vec<PathBuf>, event_proxy: EventLoopProxy<Event>) {
         return;
     }
 
+    // Watch only files.
```

Comment:
I think it would be useful to clarify what we consider "files" here. Explicitly pointing out `/dev/null` would be fine too since that should be the most relevant example.